Form footer

Danie12345 opened the pull request:
The form is added. And it is now completed.

The footer is added. And it is now completed.

Fixed a lot of styling issues regarding the width and positioning of various elements throughout the index.html document.

Prepared the page for adding future PC responsiveness.

Danie12345 committed:
Made changes in structure to scroll properly

Fixed the scrolling issue where the background would rise with the headline section.

Added grid pattern to include future sections for projects.

Danie12345 committed:
Changed grid items for un-ordered list in projects

Changed the grid to use ul and li elements rather than divs in index.html.

In index.css, made necessary changes for the grid to work properly and to align everything as it kept breaking after the structure transition.

Danie12345 committed:
Added "multi-post-stories" section and fixed bugs

Added the entire "multi-post-stories" section to the grid in index.html, with its necessary styling in index.css.

Fixed a few issues that arose with the previous grid item styling.

Danie12345 committed:
Fixed resizing bugs for some button components

Improved styling for "See Project" button and above buttons in the "multi-post-stories" section.

Danie12345 committed:
Create img-placeholder2.png

Added second background image for projects sections.

Danie12345 committed:
Finished projects section

Added all the remaining information, the buttons and styles for the projects.

Added all 6 projects.

Danie12345 committed:
Projects section complete and style bug fixes

Added an empty li element at the end of projects list in index.html.

Fixed a few style bugs introduced a few commits back in index.css.

Danie12345 committed:
Headline background bug fixed

Fixed the bug in headline background: background was too low.

Danie12345 committed:
Finished about me section

Finished organizing information on index.html.

Added and fixed many styles in index.css for the newest sections.

Danie12345 committed:
Update index.css

Changed flex box for floating for the projects' buttons.

ShingiraiBhengesa committed:
add form section

ShingiraiBhengesa committed:
Update README.md

devMahmoud committed:
add footer

Danie12345 committed:
Finished form and corrected bugs

Form is finished, though missing the action link.

Styling revamped, fixed many bugs, reorganizing code for future developments.

Danie12345 committed:
Fixed styling issue from the form alignment.

Fixed styling bug from form alignment, where smaller screens would make it move away from the vertical axis, and corrected a test left behind from a previous test in index.css.

Removed unwanted style property used for quick tests from index.html.

Danie12345 committed:
Fixed linter errors

Corrected four linter errors.

topeogunleye commented on index.html:
```diff
@@ -29,30 +29,344 @@ <h1>
+                I'm always interested in hearing about new projects, 
+                so if you'd like to chat please get in touch.
+            </h3>
+            <form action="#" method="post" class="form">
```
- [ ] [REQUIRED] I kindly suggest that you use your form action to integrate with [formspree](https://formspree.io/html/) service as this will make it possible for you to receive user submissions to your email address.

Example:
`<form action="https://formspree.io/f/{form_id}" method="post" class="form">`
            
`</form>`

topeogunleye commented on index.css:
```diff
@@ -92,52 +102,600 @@ div > span {
+  border: none;
+}
+
+.form > .submit-button {
```
- [ ] [REQUIRED] Nice job adding the button interaction for the **enabled** state :+1: . I kindly suggest that you also add the buttons interactions for the  **hover** state:

![Screenshot from 2022-03-24 00-16-40](https://user-images.githubusercontent.com/52934128/159813641-c2e9d64d-99a1-415f-827c-21e12de580b3.png)

topeogunleye reviewed (changes requested):
> # Changes Required 
> ### Hi Team,
> Nice job, you are almost there.
> 
> ![](https://camo.githubusercontent.com/34482800e1b92cb79ddca06edb4051edc407cb190f896e1e949e596aa9073851/68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966)

> 
> ## (Highlights) Good Points: 
> * You documented your work professionally. :heavy_check_mark:
> * Portfolio matches the Figma design template :heavy_check_mark: 
> * All input fields have the correct `type` attribute :heavy_check_mark: 
>
> ## (Changes Required) Aspects to improve: 
> * [ ]  Please check the replies in my inline comments.
> 
> ## [Optional] suggestions:
- [ ] [OPTIONAL] I kindly suggest you include additional descriptions of the portfolio project and its features in your readme file
>
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> **Cheers and Happy coding! :clap::clap::clap:**
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

Danie12345 committed:
Added action link for forms and button hover

Added the link for the action attribute in the form, allowing for submissions in index.html.

Modified the buttons' classes to change on hover and on click in index.css.

Danie12345 committed:
Update index.css

Changed hover and active button styles for desired ones.

Also added the disabled style for the same buttons.

AmaduKamara reviewed (approved):
# Status: Approved 🟢🟢🟢 

Hi Team,

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
